stories/Button.ts

```diff
@@ -0,0 +1,30 @@
+import '../scss/main.scss';
+
+export enum ButtonSize {
+  Small = 'small',
+  Medium = 'medium',
+  Large = 'large',
+  XLarge = 'x-large',
+}
+
+export type ButtonProps = {
+  size: ButtonSize;
+  label: string;
+  color?: string;
+  onClick: () => void;
+}
+
+export const createButton = ({
+  size,
+  color = "",
+  label,
+  onClick,
+}: ButtonProps) => {
+  const btn = document.createElement('button');
+  btn.type = 'button';
+  btn.innerText = label;
+  btn.addEventListener('click', onClick);
+
+  btn.className = ['styled-button', `styled-button--${size}`, color && `styled-button--${color}`].join(' ');
```

Comment:
I think the color here can be a select control as well? think our CDS has a limited sets of possible color? (cus now the random input of color is not being passed in to css) 
<img width="734" alt="image" src="https://github.com/compound-finance/compound-styles/assets/6854817/bc4fd515-50c6-4097-9a18-9d901a5874bd">